tests/test_po.py

```diff
@@ -55,13 +56,15 @@ class TestGTranslator(unittest.TestCase):
     def test_gtranslator(self):
         """Test all pofiles for GTranslator brokenness."""
         for f in get_pofiles():
-            with open(f, 'rb') as fd:
+            with open(f, "rb") as fd:
                 self.check_file(fd, f)
 
     def check_file(self, fd, f):
         """Test for GTranslator broken syntax."""
         for line in fd:
             if line.strip().startswith(b"#"):
                 continue
-            self.assertFalse(b"\xc2\xb7" in line,
-                 "Broken GTranslator copy/paste in %r:\n%r" % (f, line))
+            self.assertFalse(
+                b"\xc2\xb7" in line,
```

Comment:
Unrelated to this PR, but perhaps we should be checking for Unicode `\N{MIDDLE DOT}` in decoded text, not the UTF-8 encoding of it in byte strings.